recipes/r-jsonld/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set version = '2.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-jsonld
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/jsonld_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/jsonld/jsonld_{{ version }}.tar.gz
+  sha256: 9c9bcf63aad870dc2ec1eb13ca0b4c53559f845add602b546c61cc65d3a5e052
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{posix}}zip               # [win]
+  host:
+    - r-base
+    - r-v8
+    - r-curl >=2.7
+    - r-jsonlite
+  run:
+    - r-base
+    - r-v8
+    - r-curl >=2.7
+    - r-jsonlite
+
+test:
+  commands:
+    - $R -e "library('jsonld')"           # [not win]
+    - "\"%R%\" -e \"library('jsonld')\""  # [win]
+
+about:
+  home: https://github.com/ropensci/jsonld#readme (devel) https://www.w3.org/TR/json-ld (spec)
+  license: BSD_3_clause
+  summary: JSON-LD is a light-weight syntax for expressing linked data. It is primarily intended
+    for web-based programming environments, interoperable web services and for  storing
+    linked data in JSON-based databases. This package provides bindings to the  JavaScript
+    library for converting, expanding and compacting JSON-LD documents.
+  license_family: BSD
```

Comment:
```suggestion
  license_family: BSD
  license_file: LICENSE
```

Reply:
Thanks for the review. Now updated with the help of https://github.com/conda/conda-build/pull/3616